stock-usecase/build.gradle.kts

```diff
@@ -92,6 +92,7 @@ dependencies {
     implementation(project(":dhis_android_analytics"))
 
     implementation(libs.bundles.stock.implementation)
+    implementation(project(":aggregates"))
```

Comment:
Does the stock-usecase module need to know about the aggregate module

Reply:
I think it should not